app/views/layouts/application.html.slim

```diff
@@ -21,9 +21,8 @@ html lang="#{I18n.locale}" class='no-js'
     <!--[if IE 8]>
       = javascript_include_tag 'es5-shim.min'
     <![endif]-->
-    == javascript_include_tag 'misc/i18n-strings'
-    - unless controller_name == 'home'
-      == javascript_include_tag 'application'
+    == javascript_include_tag 'i18n-strings'
+    == javascript_pack_tag 'application'
```

Comment:
We don't need the controller conditional anymore?

Reply:
I don't think a "home" controller exists anymore?

Reply:
heh. you're right!